hudi-spark/pom.xml

```diff
@@ -213,9 +213,9 @@
 
     <!-- Spark (Packages) -->
     <dependency>
-      <groupId>com.databricks</groupId>
+      <groupId>org.apache.spark</groupId>
       <artifactId>spark-avro_2.11</artifactId>
-      <version>4.0.0</version>
+      <scope>provided</scope>
```

Comment:
I understand we can't bundle this since its tied to a spark version now.  `spark-avro` is still a package and the user must explicitly include using `--packages` ? So, if I upgrade hudi in the next release, then as a user I need to change something? Should/how do we document this?

Reply:
Yeah this will be an additional Jar, the user would have to pass while starting the spark-shell. We would have to document it. I don't see any documentation for  `spark.serializer=org.apache.spark.serializer.KryoSerializer` either which is also a pre-requisite right. Shall we update it in the README ?

Reply:
`KryoSerializer` command line is provided on the quickstart page. Adding it there could be a good thing. Do a follow on update?